Galaxus.Functional/(Option)/OptionExtensions.cs

```diff
@@ -239,15 +239,23 @@ public static async Task<T> UnwrapOrElseAsync<T>(this Task<Option<T>> self, Func
             return await fallback();
         }
 
+        #endregion
+    }
+
+    /// <summary>
+    ///     Additional extensions for the option type, providing some quality of life-shorthands.
+    /// </summary>
+    public static class OptionExtensions2
```

Comment:
Wow, TIL that type constraints are not enough to ensure two signatures are different.

I feel having `OptionExtensions2` as a name is a bit *meh*, but at the same time - the only grouping that'd make sense is putting both `ToNullable` in the same class, which again does not work. Dangit!

What about `ReferenceTypeOptionExtensions`? Most people won't see the name of the static class, but it is possible to call it like that.

Reply:
I'll put the other `.ToNullable()` in a third class called `ValueTypeOptionExtensions`. Then it is at least symetric. What do you think about that?

Reply:
Ooooh I love that! Great idea!

Reply:
done